src/components/modals/LoginModal.tsx

```diff
@@ -39,24 +41,17 @@ export default function LoginModal({
 
   const isLoading = loadingRequestToken || hasStartCaptcha
 
-  const onSubmit = async (e: SyntheticEvent) => {
-    e.preventDefault()
-    beforeLogin?.()
-    if (await login(privateKey)) {
-      afterLogin?.()
-      setPrivateKey('')
-      props.closeModal()
-    } else {
-      toast.custom((t) => (
```

Comment:
pls add this back and do this if `loginRes` is false

Reply:
Must have gotten removed unawaredly